connectors/yarpc/yarpc_test.go

```diff
@@ -181,7 +182,7 @@ func TestYaRPCClient_Read(t *testing.T) {
 	}
 
 	// we expect a single call to Read, and we return back two fields, f1 which is in the typemap and another field that is not
-	mockedClient.EXPECT().Read(ctx, readRequest).Return(&drpc.ReadResponse{drpc.FieldValueMap{
+	mockedClient.EXPECT().Read(ctx, readRequest, gomock.Any()).Return(&drpc.ReadResponse{drpc.FieldValueMap{
```

Comment:
Use something more specific than gomock.Any() to verify that the correct version is being passed